python/ml_estim_incr_signal_data.py

```diff
@@ -11,12 +11,11 @@
 xtrue[79] = 1
 xtrue = np.cumsum(xtrue)
 
-# pass the increasing input through a moving-average filter 
+# pass the increasing input through a moving-average filter
 # and add Gaussian noise
 h = np.array([1, -0.85, 0.7, -0.3])
 k = h.size
 yhat = np.convolve(h,xtrue)
 y = yhat[0:-3].reshape(N,1) + np.random.randn(N,1)
 
-xtrue = np.asmatrix(xtrue.reshape(N,1))
-y = np.asmatrix(y)
+xtrue = xtrue.reshape(N,1)
```

Comment:
I wonder if we can get rid of all the reshapes, and just have `h, y, xtrue` all be one dimensional arrays?

Reply:
Good point. It has been updated.
Note that the solution file needs to be updated as well.
See out of channel comments.